models/user.js

```diff
@@ -16,25 +16,21 @@ const UserSchema = new Schema({
     type: String,
     required: true,
   },
-  displayName: {
-    type: String,
-  },
+  displayName: String,
   profileImg: ImageSchema,
-  realName: {
-    type: String,
-  },
-  birthDate: {
-    type: String,
-  },
-  country: {
-    type: String,
-  },
-  location: {
-    type: String,
-  },
-  bio: {
-    type: String,
+  realName: String,
+  birthDate: String,
+  country: String,
+  location: String,
+  bio: String,
+  bestScores: {
+    veryEasy: Number,
+    easy: Number,
+    medium: Number,
+    hard: Number,
+    veryHard: Number,
   },
+  customSessionHighScores: {},
```

Comment:
This is not referenced anywhere in this Pull Request, not sure what this is for.

Reply:
I put this in there since I am still considering perhaps having some custom play sessions in the future. I guess it is unnecessary now, and may or may not be necessary later, but I ended up putting it there mostly to figure out how to better organize scores in the user model. Can remove it if that would be better.